apps/frontend/src/pages/overview/libs/components/courses/courses.tsx

```diff
@@ -0,0 +1,27 @@
+import { Course } from "~/libs/components/components.js";
+import { type CourseDto } from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	courses: CourseDto[];
+};
+
+const Courses: React.FC<Properties> = ({ courses }: Properties) => {
+	return (
+		<div className={styles["container"]}>
+			<h2 className={styles["title"]}>Courses</h2>
+			<ul className={styles["list"]}>
+				{courses.map((course: CourseDto) => {
```

Comment:
```suggestion
				{courses.map((course) => {
```
Do we need to specify this type again? We already have it from Properties